packages/doc-contribs/pr-antora-content-guidelines-checker/src/validation.ts

```diff
@@ -66,3 +66,8 @@ export interface ActionResult {
   status: string;
   results: ValidationResult[];
 }
+
+export enum AvailableSteps {
+  ATTRIBUTES_CHECKING_STEP = "attributes-checking-step",
```

Comment:
**suggestion:** The enum is about Steps, so I would remove the "step" suffix in enum entries as it seems redundant.

In addition, the TypeScript convention for enums is (https://www.typescriptlang.org/docs/handbook/enums.html#handbook-content)
- enum name: CamelCase and singular
- enum entries: CamelCase and singular

Let's talk about this in person, I don't really care if we don't follow this convention